frames/section.lua

```diff
@@ -279,6 +286,39 @@ function sectionFrame:_DoCreate()
     onTitleClickOrDrop(s)
   end)
 
+  title:SetScript("OnClick", function(_, e)
```

Comment:
This is incorrect -- Setting script twice will overwrite the click handler set above this. Move all of this into `onTitleClickOrDrop` and we'll review it then.